src/client/render/core.cpp

```diff
@@ -84,8 +84,12 @@ void RenderingCore::draw3D()
 void RenderingCore::drawHUD()
 {
 	if (show_hud) {
-		if (draw_crosshair)
+		if (draw_crosshair) {
 			hud->drawCrosshair();
+
+			if (hud->can_draw_selectionindicator)
+				hud->drawSelectionIndicator();
```

Comment:
I suggest to move this into `Hud::drawCrosshair`. (So that the selection indicator is the crosshair.)

Reply:
Done